pulse2percept/stimuli.py

```diff
@@ -104,14 +116,120 @@ def __init__(self, ptype, pdur, tsample, interphase_dur=0):
         utils.TimeSeries.__init__(self, tsample, pulse)
 
 
+class PulseTrain(utils.TimeSeries):
+
+    def __init__(self, tsample, freq=20, amp=20, dur=0.5, delay=0,
+                 pulse_dur=0.45 / 1000, interphase_dur=0.45 / 1000,
+                 pulsetype='cathodicfirst',
+                 pulseorder='pulsefirst'):
+        """A train of biphasic pulses
+
+        tsample : float
+            Sampling interval in seconds parameters, use TemporalModel.tsample.
+        ----------
+        optional parameters
```

Comment:
This docstring looks a bit weird. Could you please change this into a "Parameters" block, and indicate all the optional parameters, and their default values?